lua/nvim-paredit/lang/init.lua

```diff
@@ -10,16 +16,28 @@ local function keys(tbl)
   return result
 end
 
-return {
-  get_language_api = function()
-    return langs[vim.bo.filetype]
-  end,
+function M.get_language_api()
+  return langs[vim.bo.filetype]
+end
 
-  add_language_extension = function(filetype, api)
-    langs[filetype] = api
-  end,
+function M.add_language_extension(filetype, api)
+  langs[filetype] = api
+end
 
-  filetypes = function()
-    return keys(langs)
-  end,
-}
+function M.filetypes()
+  return keys(langs)
+end
+
+function M.is_whitespace_under_cursor()
+  local cursor = vim.api.nvim_win_get_cursor(0)
+  local lang = M.get_language_api()
+  cursor = { cursor[1] - 1, cursor[2] }
+
+  local char_under_cursor = vim.api.nvim_buf_get_text(0, cursor[1], cursor[2], cursor[1], cursor[2] + 1, {})
+  return common.included_in_table(
+    lang.whitespace_chars or default_whitespace_chars,
+    char_under_cursor[1]
+  ) or char_under_cursor[1] == ""
+end
```

Comment:
I think this fn should live in utils somewhere rather

Reply:
agree